src/mlpack/core/cv/k_fold_cv_impl.hpp

```diff
@@ -270,7 +270,7 @@ double KFoldCV<MLAlgorithm,
     return 0.0;
   }
 
-  return arma::mean(evaluations.elem(arma::find_finite(evaluations)));
+  return mean(evaluations.elem(arma::find_finite(evaluations)));
```

Comment:
```suggestion
  return mean(evaluations.elem(find_finite(evaluations)));
```
Probably can remove that one too?

Reply:
Sounds good, I was focused on RNNs so I missed a bunch of these. Ill fix them when I'm free.